lib/Spotify.py

```diff
@@ -20,8 +20,33 @@ def get_playlists(self):
         returns: list of playlist objects
         '''
         playlists = self._spotify.current_user_playlists()
+        playlists["items"].append(self.get_playlist("saved"))
         return playlists
 
+    def get_playlist(self, id):
+        '''
+        Gets information about a given playlist
+        '''
+        print(id)
+        if(id == "saved"):
+            playlist = self._spotify.current_user_saved_tracks()
+
+            playlist["collaborative"] = False
+            playlist["description"] = "Saved tracks"
+            playlist["name"] = "Saved tracks"
```

Comment:
So so nit-picky but just a change from "Saved tracks" to "Saved Tracks" (Capitalize tracks)